src/QMCWaveFunctions/Jastrow/RadialJastrowBuilder.cpp

```diff
@@ -223,7 +224,7 @@ std::unique_ptr<WaveFunctionComponent> RadialJastrowBuilder::createJ2(xmlNodePtr
 #if OHMMS_DIM == 1
         RealType dim_factor = 1.0 / (OHMMS_DIM + 1);
 #else
-        RealType dim_factor = (ia == ib) ? 1.0 / (OHMMS_DIM + 1) : 1.0 / (OHMMS_DIM - 1);
+        RealType dim_factor = (ia == ib) ? 0.5 / (ndim - 1) : 1.0 / (ndim - 1);
```

Comment:
For 2 dimensions, the factor for the case ia==ib changes from 1/3 to 1/2.  Is that change intended?

Reply:
Yes, this is an intentional change.
The same-spin cusp should always be half of that of the opposite-spin cusp.
Did I miss something?

Reply:
The previous code was added in #2806 , and it references equation 9 of https://arxiv.org/abs/2003.06506 (and footnote 45 for the 1-D case).  According to that formula, it looks like the same-spin cusp is half of the opposite spin cusp for d=3, but not for d=2.

Reply:
Yes, you are right. I must have made a mistake in my own derivation.
Thanks for catching this!